src/main/java/uk/gov/hmcts/reform/civil/constants/ResponseJourneyConstantsLRSpec.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.civil.constants;
+
+public final class ResponseJourneyConstantsLRSpec {
```

Comment:
We can may be have a common file for Claim issue and response, so we can name it accordingly.
`SpecJourneyConstantLRSpec.java` may be
Also is there a reason for not having this as interface?